rotkehlchen/assets/asset.py

```diff
@@ -563,6 +563,10 @@ def to_dict(self) -> dict[str, Any]:
             'underlying_tokens': underlying_tokens,
         }
 
+    @property
```

Comment:
if you do it like this there is no need to put the `@property` on top. Leave it a function. The only reason to make it a propery is to shadow the decimals, or turn main decimals to something like `_decimals` and have a property at `decimals`.